pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -51,6 +51,11 @@ func (a Adapter) Push(path string, ignoredFiles []string, forceBuild bool, globE
 		return errors.Wrap(err, "unable to create or update component")
 	}
 
+	_, err = a.Client.WaitForDeploymetRollout(a.ComponentName)
```

Comment:
typo: missing the `n` in `WaitForDeploymetRollout`

Reply:
😇 ups.
Fixed. Thx